prometheus-exporters/elastic-query-exporter/files/exporter.cfg

```diff
@@ -156,3 +155,24 @@ QueryJson = {
           }
         }
     }
+
+[query_elasticsearch_octobus_vmdown]
+QueryIntervalSecs = 300
+QueryTimeoutSecs = 15
+QueryIndices = c0001_log-*
+QueryJson = {
+    "query": {
+      "bool": {
+        "must": [
+          { "exists": { "field": "node.nodename" }},
+          { "match_phrase": { "message": "An operation required the virtual machine to quiesce and the virtual machine was unable to continue running." }}
+        ],
+        "filter": [
+          { "range": { "@timestamp": { "gte": "now-1h" }}}
+        ]
+      }
+    },
+    "aggs": {
+      "hostsystem": { "terms": { "field": "node.nodename.keyword" }}
```

Comment:
can't we just use `message` as field?

Reply:
Unfortunately the mapped keyword field doesn't behave consistently on ElasticDB side. In the sample alert I was testing with this field was "missing" and the aggregate returned no results.

Reply:
I just don't think that only pointing out the host is a huge help, if there are dozens of powered off VMs.

Could you please reach out to OCB Team, why this field behaves differently?

Reply:
Yes, I'm on it. I'd suggest to merge this to test the current state in QA in parallel.